pydantic/utils.py

```diff
@@ -413,19 +413,20 @@ class GetterDict(Representation):
     We can't inherit from Mapping[str, Any] because it upsets cython so we have to implement all methods ourselves.
     """
 
-    __slots__ = ('_obj',)
+    __slots__ = ('_obj', '_kwargs')
 
-    def __init__(self, obj: Any):
+    def __init__(self, obj: Any, **kwargs: Any):
```

Comment:
I adjusted this slightly and typed `kwargs` as an `Optional[Dict[str, Any]]` instead, since anyone who might've subclassed `GetterDict` would now be missing the new `kwargs` param (if they overrode the `__init__` method). I can change it back if need be.